week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,3 +1,25 @@
+function setAlarm() {
+
+    var timmer = document.getElementById("timeRemaining")
+    var counter = document.getElementById("alarmSet").value
+    console.log(counter);
+    timmer.innerHTML = "00:" + counter
+
+
+    setInterval(() => {
+        if (counter > 0) {
+            counter--
+            timmer.innerHTML = "00:" + counter
```

Comment:
To make this extra pretty, can you make this so that it always displays two digits? i.e. `00:09` rather than `00:9`? And maybe even so that if you have more than one minute, it displays `01:10` rather than `00:70`?